doc/introduction/compiling_circuits.rst

```diff
@@ -15,6 +15,12 @@ or replace a large circuit by a number of smaller circuits.
 Compilation functionality is mostly designed as **transforms**, which you can read up on in the
 section on :doc:`inspecting circuits </introduction/inspecting_circuits>`.
 
+PennyLane also supports experimental just-in-time compilation, via
```

Comment:
> In addition to quantum circuit transformations

This phrase from below might help smooth the transition between the two compilation concepts.

Reply:
```suggestion
In addition to quantum circuit transforms, PennyLane also
supports experimental just-in-time compilation, via
```